soc/arm/nxp_imx/rt/soc.c

```diff
@@ -50,7 +50,7 @@ const clock_usb_pll_config_t usb1PllConfig = {
 #ifdef CONFIG_INIT_ENET_PLL
 /* ENET PLL configuration for RUN mode */
 const clock_enet_pll_config_t ethPllConfig = {
-#if defined(CONFIG_SOC_MIMXRT1021) || defined(CONFIG_SOC_MIMXRT1015) || defined(CONFIG_SOC_MIMXRT1011)
+#if defined(CONFIG_SOC_MIMXRT1021) || defined(CONFIG_SOC_MIMXRT1015) || defined(CONFIG_SOC_MIMXRT1011) || defined(CONFIG_SOC_MIMXRT1024)
```

Comment:
```suggestion
#if defined(CONFIG_SOC_MIMXRT1021) || \
      defined(CONFIG_SOC_MIMXRT1015) || \
      defined(CONFIG_SOC_MIMXRT1011) || \
      defined(CONFIG_SOC_MIMXRT1024)
```
Just code style, helping when more socs coming.